client/src/components/LineMovieItem.jsx

```diff
@@ -0,0 +1,36 @@
+import * as PropTypes from "prop-types";
+import styles from "../assets/styles/components/line-movie-item.module.css";
+
+function LineMovieItem({ title, posterPath, voteAverage, voteCount }) {
+  return (
+    <div className={styles.lineMovieItem}>
```

Comment:
Retirer les { } et les remplacer par " " puis enlever styles.
Pour toutes les itérations

Reply:
Done.